drivers/adodb-mysqli.inc.php

```diff
@@ -157,16 +157,64 @@ function SetTransactionMode($transaction_mode)
 	 * Parameter must be one of the constants listed in mysqli_options().
 	 * @see https://www.php.net/manual/en/mysqli.options.php
 	 *
-	 * @param int    $parameter The parameter to set
-	 * @param string $value     The value of the parameter
+	 * @param string|int $parameter The parameter to set
+	 * @param string|int|array $value The value of the parameter
 	 *
 	 * @return bool
 	 */
 	public function setConnectionParameter($parameter, $value) {
-		if(!is_numeric($parameter)) {
+
+		// Special case for setting SSL cert values.
+		if ("sslcert" === $parameter && is_array($value)) {
+
+			if (true === isset($value["ssl_key"])) {
```

Comment:
isset() returns a bool, so `true ===` is not necessary

I also suggest naming the parameter just `ssl`, instead of `sslcert` as some of the values are not about the certificate (e.g. cipher). 

Do we need the `ssl_` prefix in the array key ? It feels redundant.